pkg/querier/tripperware/instantquery/instant_query.go

```diff
@@ -324,20 +324,30 @@ func vectorMerge(req tripperware.Request, resps []*PrometheusInstantQueryRespons
 			if s == nil {
 				continue
 			}
-			metric := string(cortexpb.FromLabelAdaptersToLabels(sample.Labels).Bytes(buf))
+			metric := cortexpb.FromLabelAdaptersToLabels(sample.Labels).String()
```

Comment:
Can we still keep the `Bytes`? It should has better performance than `String`.
https://github.com/cortexproject/cortex/pull/5095/files

Reply:
In Thanos we haven't ported this change.